docs/architecture/tablets.rst

```diff
@@ -0,0 +1,92 @@
+=========================================
+Data Distribution  with Tablets
+=========================================
+
+A ScyllaDB cluster is a group of interconnected nodes - on-premises servers or 
+virtual servers (public cloud instances). The data of the entire cluster has to
+be distributed as evenly as possible across those nodes.
+
+ScyllaDB is designed to ensure a balanced distribution of data when adding new
+nodes (scaling out) by storing data in tablets. When you scale out, tablets are
+moved between the nodes to keep the same number on each node.
+
+This article explains the concept of tablets and how they let you scale your
+cluster quickly and seamlessly.
+
+Data Distribution
+-------------------
+
+ScyllaDB distributes data by splitting tables into tablets. Each tablet has 
+its replicas on different nodes, depending on the RF (replication factor). Each
+row of a table is mapped to a single tablet in a deterministic way. When you
+query or update the data, ScyllaDB can quickly identify the tablet that stores
+the relevant row. 
+
+The following example shows a 3-node cluster with a replication factor (RF) of
+3. The data is stored in a table (Table 1) with two rows. Both rows are mapped
+to one tablet (T1) with replicas on all three nodes.
+
+.. TODO - add the image
+
+When a tablet reaches a threshold size (measured in the number of rows),
+the tablet splits into two.
+
+.. TODO - add an image with tablet splitting
+
+Load Balancing
+==================
+
+ScyllaDB autonomously moves tablets to balance the load based on table usage. 
+This process is managed by a load balancer mechanism and happens independently
+of the administrator or node operations such as bootstrap or decommission.
+The tablet load balancer decides where to migrate the tablets, either within
+the same node to balance the shards or across the nodes to balance the global
+load in the cluster. 
+
+The load-balancing process takes place in the background and is performed
+without any service interruption.
+
+Scaling Out
+=============
+
+A tablet can be dynamically migrated to either an existing node or a new node –
+on bootstrap. Tablets make scaling out fast and efficient, as the new node
+starts taking part of the load even before the bootstrap process completes.
```

Comment:
Before the bootstrap is complete or before migration is complete?

Reply:
Tablets are migrated to the new node only post-bootstrap, in the background. Such bootstrap is short with tablets, as it only migrates vnode-based keyspaces.